.github/workflows/ci.yaml

```diff
@@ -16,8 +16,57 @@ on:
     - cron: "0 4 * * *"
 
 jobs:
+
+  populate-build-cache:
+    name: "Populate build cache"
```

Comment:
I would even go as far as this is just `Build everything using cabal`

The other jobs then are only about running the individual test suites and benchmarks?

Reply:
Good idea for an other P.R.